crates/state/Cargo.toml

```diff
@@ -14,8 +14,12 @@ publish = false
 anyhow = "1"
 blake3 = "1"
 futures = "0.3"
-sqlx = { version = "0.5", features = ["runtime-tokio-rustls", "sqlite", "macros"] }
+# https://github.com/launchbadge/sqlx/issues/713
```

Comment:
We discussed over Slack that it would probably be better to revisit in the future whether we could avoid using `Actix` and use `Tokio`

Reply:
https://github.com/spacemeshos/svm/issues/365 indeed!